pkg/apis/scheduling/types.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduling
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient=true
+// +nonNamespaced=true
+
+// PriorityClass defines the mapping from a priority class name to the priority
+// integer value. The value can be any valid integer.
+type PriorityClass struct {
+	metav1.TypeMeta
+	// Standard object metadata; More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata.
+	// +optional
+	metav1.ObjectMeta
+
+	// The value of this priority class. This is the actual priority that pods
+	// receive when they have the name of this class in their pod spec.
+	Value int32
+
+	// GlobalDefault specifies whether this PriorityClass should be considered as
+	// the default priority for pods that do not have any priority class and their
+	// namespace do not have any other default priority.
```

Comment:
The priority admission controller is the component that uses "PriorityClass" to resolve priority class name of a pod to the integer value of the priority. The same admission controller enforces that there will be only one global default. If the admission controller is disabled by a user, there won't be any priority name resolution and global default will be meaningless. So, in the absence of the admission controller, one can add more than one priority class with global default and they will be unused.